x/incentive/types/params.go

```diff
@@ -28,6 +29,7 @@ func NewParams() Params {
 		StakeIncentives:     []IncentiveInfo(nil),
 		CommunityTax:        sdk.NewDecWithPrec(2, 2), // 2%
 		WithdrawAddrEnabled: true,
+		RewardPercentForLps: sdk.NewDecWithPrec(65, 2),
```

Comment:
Can we update the name to RewardPortionForLps? Since it's not percent but a decimal value, I think the word portion would make more sense.